src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
 // write code here
+const sortList = document.querySelector('thead');
+const bodyList = document.querySelector('tbody');
+const sorted = [...bodyList.rows];
```

Comment:
```suggestion
const rows = [...bodyList.rows];
```
Они еще не отсортированы